bridge/src/commonMain/kotlin/ru/herobrine1st/matrix/bridge/telegram/TelegramWorker.kt

```diff
@@ -50,97 +52,138 @@ class TelegramWorker(
     ) {
         when (event) {
             is ClientEvent.RoomEvent.MessageEvent<*> -> {
-                when (val content = event.content) {
-                    is RoomMessageEventContent -> {
-                        val bot = getBot(actorId)
-                        val result = when (content) {
-                            is RoomMessageEventContent.TextBased.Text -> {
-                                val textContent = content.body
-
-                                withContext(Dispatchers.IO) {
-                                    bot.sendMessage(
-                                        chatId = roomId,
-                                        text = "<${event.sender}>: $textContent",
-                                    )
-                                }
-                            }
+                val contentRaw = event.content
+                if (contentRaw !is RoomMessageEventContent) return
```

Comment:
You should not change that because there would be a branch with `RedactionEventContent`

```suggestion
                when (val contentRaw = event.content) {
                    is RoomMessageEventContent -> {
```